devel/to-f90/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
```

Comment:
This is our standard modeline, and it's great that you're using this, but note that it states that indentations should use multiples of 4 spaces, but you've indented using tabs. Please convert the tabs to spaces.

Reply:
OK